ratatoskr/app/templates/app/components/schedule_card.html

```diff
@@ -1,8 +1,11 @@
-<div class="card">
+<div class="card card-hover-shadow">
     <div class="card-body">
-        <a href="{% url 'schedule' schedule.id %}">
-            <h5 class="card-title">{{ schedule.owner.socialaccount_set.all.0.extra_data.name }}</h5>
-        </a>    
-        <h6 class="card-subtitle mb-2 text-muted">{{ schedule.name }}</h6>
+        <a href="{% url 'user-schedules' schedule.owner.id %}" class="text-muted text-decoration-none">
+            <div class="d-flex flex-row align-items-center">
+                <img class="me-2 rounded" src="{{ schedule.owner.socialaccount_set.all.0.get_avatar_url }}" alt="{{ schedule.owner.get_full_name }}'s profile picture" style="width: 20px">
```

Comment:
I had no idea that get_full_name was a thing. I could have saved some time typing that instead of the stupidly long `user.socialaccount_set.all.0.extra_data.name`...